### PR TITLE
documentation: add readme for measurement-server

### PR DESCRIPTION
**Pull request by Ruoyu-y**
* add readme for the service

### CODE REVIEW

File: .github/cspell.json [hunk 1]
```diff
@@ -5,10 +5,19 @@
...
     "ignorewords": [],
+    "ignoreRegExpList": [
+        "/```[\\s\\S]*?```/g",
+        "/`[\\s\\S]*?`/g"
+    ],
     "import": [],
     "dictionarydefinitions": [],
     "dictionaries": [],
     "words": [
-        "ccnp"
```
Comment (hwang37) [inline .github/cspell.json:12]: Please use alphabet sequence to sort dictionary words.

File: service/measurement-server/README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+Measurements are key assets in the confidential computing world for user to verify and determine the trustworthiness of the environment and its underlying platform.
+Users can utilize different kinds of measurements to monitor or validate the operations performed within the platform.
+
+This service (which is actually a GRPC server) will provide support to fetch the measurements for confidential cloud native environments, including TDX RTMR measurements, TPM PCR measurements and TEE reports.
+
+
+
+## Introduction
+
+This service provides functionality to fetch measurements for confidential cloud native environments, both platform level(`PAAS` option) and container level (`SAAS` option). Using this service, user can fetch different
```
Comment (hwang37) [inline service/measurement-server/README.md:12]: Suggest to use "PaaS" and "SaaS"

File: service/measurement-server/README.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+To select different level and category of measurements, user shall send out request with different settings.
+User can find sample request in the Testing section.
+
+The collected measurements are returned as json string to the client.
+
+
+
+## Installation
+
+The Measurement service can be deployed as either DaemonSet or sidecar in different user scenarios.
```
Comment (hwang37) [inline service/measurement-server/README.md:54]: according to different user scenario???

File: service/measurement-server/README.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+
+### Prerequisite
+
+To simplify the deployment process, we use Helm as the basic tool for deployment. Please install Helm following the [Helm official guide](https://helm.sh/docs/intro/install/).
```
Comment (hwang37) [inline service/measurement-server/README.md:59]: by following

File: service/measurement-server/README.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+### Build docker image
+
+The dockerfile for the service can be found under `container/measurement-server` directory. Use the following command to build the image:
+
+```
+cd ../..
+docker build -t ccnp_measurement_server:0.1 -f container/measurement-server/Dockerfile .
+```
+> Note: if you are using containerd as the default runtime for kubernetes. Please remember to use the following command to import the image into containerd:
```
Comment (hwang37) [inline service/measurement-server/README.md:70]: if you are using containerd as the default runtime for kubernetes, don't forget to use the following commands to import the image into containerd first:

File: service/measurement-server/README.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+### User application deployment
+
+Make sure that the user application requests such resource in the yaml file to take use of sdk to contact the measurement server: `tdx.intel.com/tdx-guest: 1`.
+
+
+
+## Testing
+
+One can play with service on host from the source code following the steps below:
```
Comment (hwang37) [inline service/measurement-server/README.md:100]: User can play with service on host by following following the steps below:

File: .github/cspell.json (hunk 1 above)
Comment (hwang37) [reply .github/cspell.json:12]: Add the following words if there's not.
GRPC
PaaS
SaaS
RTMR
grpcurl

File: service/measurement-server/README.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
+# Service: CCNP Measurement Server
+
+Measurements are key assets in the confidential computing world for user to verify and determine the trustworthiness of the environment and its underlying platform.
+Users can utilize different kinds of measurements to monitor or validate the operations performed within the platform.
+
+This service (which is actually a GRPC server) will provide support to fetch the measurements for confidential cloud native environments, including TDX RTMR measurements, TPM PCR measurements and TEE reports.
```
Comment (hairongchen) [inline service/measurement-server/README.md:6]: support => change to "interface" or "API" to be more specific?

File: service/measurement-server/README.md (hunk 2 above)
Comment (Ruoyu-y) [reply service/measurement-server/README.md:12]: These two are using the same capital words as protobuf defined. May i still use the capital ones?

File: service/measurement-server/README.md (hunk 3 above)
Comment (Ruoyu-y) [reply service/measurement-server/README.md:54]: The user scenario here means if they are using CCNP for confidential cluster(deploy as Daemonset) or confidential containers(deploy as sidecar), is it appropriate to use 'user scenario' to differentiate such two cases?

File: .github/cspell.json (hunk 1 above)
Comment (Ruoyu-y) [reply .github/cspell.json:12]: Done.  BTW, words that are used inside code blocks will be ignored for cspell scan.

File: service/measurement-server/README.md (hunk 2 above)
Comment (hwang37) [reply service/measurement-server/README.md:12]: sure, not a big deal

File: service/measurement-server/README.md (hunk 3 above)
Comment (hwang37) [reply service/measurement-server/README.md:54]: fine

**Comment by Ruoyu-y** [comment]
add extra explanation on deployment based on yaml files

**Review by kenplusplus — APPROVED** [review]
LGTM

**Pull request merged**